tests/test_dataset.py

```diff
@@ -31,19 +32,21 @@ def test_dataset(self):
                 "meta": {
                     "great_expectations.__version__": ge.__version__
                 },
-                "expectations" : [{
-                    "expectation_type" : "expect_column_to_exist",
-                    "kwargs" : { "column" : "x", 'result_format': 'BASIC'},
-                    'success_on_last_run': True
-                },{
-                    "expectation_type" : "expect_column_to_exist",
-                    "kwargs" : { "column" : "y", 'result_format': 'BASIC'},
-                    'success_on_last_run': True
-                },{
-                    "expectation_type" : "expect_column_to_exist",
-                    "kwargs" : { "column" : "z", 'result_format': 'BASIC'},
-                    'success_on_last_run': True
-                }]
+                "expectations" : []
+                # No longer expect autoinspection 20180920
```

Comment:
I think that leaving this in is a smart move. I wonder if we should include a link to this PR.

Also, is there other messaging that we'll need to do before rolling this out? Some of our docs and videos could be out of date, for example.